frontend/src/pages/projects/screens/projects/EmptyProjects.tsx

```diff
@@ -20,7 +24,7 @@ const EmptyProjects: React.FC = () => (
     <EmptyStateBody>
       To get started, create a data science project or launch a notebook with Jupyter.
```

Comment:
@kywalker-rh What do we do here if you cannot create a project. Say ask your admin?

Can you create a project?
- yes: "To get started, create a data science project or launch a notebook with Jupyter."
- no: "To get started, ask your {PRODUCT_NAME} admin for a data science project or launch a notebook with Jupyter."

Reply:
oh yeah!

And what it looks like when the self provis is given a namespace by the admins
![image](https://user-images.githubusercontent.com/8126518/225133083-32d396cd-036a-45ac-aaf2-d864952e52b1.png)

Reply:
I repeated the same flow, and then I was able to delete the project, should the user be able to do that? @andrewballantyne @DaoDaoNoCode

Reply:
@lucferbux That depends. If you setup your user to use the project as an ADMIN, yes, they have powers over it... including sharing. If you revoked self provisioning from your user, then gave them a project under EDIT, no -- I suspect K8s should stop that.

You gotta remember, we don't "do" permissions. We are only sharing the permissions values from OpenShift Console. Heck, we aren't even *setting* the permissions on our side. So if K8s let you do it, you gave too much permissions.

This PR doesn't grant or revoke access to things, it just checks the access.